desktop/install/ubuntu.md

```diff
@@ -51,10 +51,21 @@ Recommended approach to install Docker Desktop on Ubuntu:
 2. Download latest [DEB package](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.19.0-amd64.deb?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64).
 
 3. Install the package with apt as follows:
-
+   Go to the directory containing the deb package
+      
+   > **Note**
+   >
+   > `pass` may also need to be installed to install the deb package.
+   > 
+   > To install `pass`:
+   >
+   > ```console
+   > $ sudo apt install pass
+   > ```
+   
    ```console
    $ sudo apt-get update
-   $ sudo apt-get install ./docker-desktop-<version>-<arch>.deb
+   $ sudo dpkg -i docker-desktop-<version>-<arch>.deb
```

Comment:
@Huell-Howitzer As far as I can tell and know apt and d[kg both work, what are the advantages of using dpkg instead here?

Reply:
dpkg can be used to install individual Debian packages from local files using the command dpkg -i <package.deb>. This allows you to manually install packages, even if they are not available in the repositories. On the other hand, apt-get relies on the repositories and cannot directly install local packages.

Reply:
I think both should work;

```
curl -O https://download.docker.com/linux/ubuntu/dists/jammy/pool/stable/arm64/docker-ce-cli_24.0.1-1~ubuntu.22.04~jammy_arm64.deb

 apt-get install ./docker-ce-cli_24.0.1-1~ubuntu.22.04~jammy_arm64.deb
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'docker-ce-cli' instead of './docker-ce-cli_24.0.1-1~ubuntu.22.04~jammy_arm64.deb'
Recommended packages:
  docker-buildx-plugin docker-compose-plugin
The following NEW packages will be installed:
  docker-ce-cli
0 upgraded, 1 newly installed, 0 to remove and 15 not upgraded.
Need to get 0 B/12.0 MB of archives.
After this operation, 35.0 MB of additional disk space will be used.
Get:1 /docker-ce-cli_24.0.1-1~ubuntu.22.04~jammy_arm64.deb docker-ce-cli arm64 5:24.0.1-1~ubuntu.22.04~jammy [12.0 MB]
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package docker-ce-cli.
(Reading database ... 4975 files and directories currently installed.)
Preparing to unpack .../docker-ce-cli_24.0.1-1~ubuntu.22.04~jammy_arm64.deb ...
Unpacking docker-ce-cli (5:24.0.1-1~ubuntu.22.04~jammy) ...
Setting up docker-ce-cli (5:24.0.1-1~ubuntu.22.04~jammy) ...

docker --version
Docker version 24.0.1, build 6802122
```

Reply:
I'm not sure why you didn't get this error:

```
N: Download is performed unsandboxed as root, as file '/home/user/Downloads/docker-desktop.deb' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)
```

But that's what I got. It actually mentions that will happen in the instructions but I must have glossed over that. It does say that it is benign though. 


However, if I remember correctly, I got the error and then reran with dpkg  which does not require the package manager so it does not encounter the permission issue. On the other hand though, it doesn't handle any dependency resolution automatically.


If you want to keep the current version, please let me know if there's something that I need to do on my end as I've never actually done a pull request before.

Reply:
@Huell-Howitzer and @thaJeztah I'll try again, but I'm running on an Apple Silicon mac, so have to use a VM with arm versions of Linux, so I may get some odd dependency issues. Will get back to you.